moveit_ros/integration_tests/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(moveit_ros_integration_tests)
```

Comment:
This isn't a big deal, but don't you think its redundant - what other type of integration test could we have? We could have non-ROS unit tests, but ROS is what MoveIt! uses for all high level integration